src/frontend/js/order.js

```diff
@@ -23,4 +23,45 @@ $(function() {
             }
         });
     });
+
+    // Place batch orders
+    // --
+    // Init multidatepicker on input directly would be simpler,
+    // but a glitch would appear, so I init multidatespicker on an empty div#delivery_dates
+    // and link it to an HTML input#id_delivery_dates using altField option
+    // @see: https://github.com/dubrox/Multiple-Dates-Picker-for-jQuery-UI/issues/162
+    $('#delivery_dates').multiDatesPicker({
+        dateFormat: "yy-mm-dd",
+        separator: "|",
+        minDate: 0,
+        numberOfMonths: 2,
+        altField: '#id_delivery_dates'
+    });
+    $('#id_delivery_dates').hide();
+
+    $('#id_client').change(function() {
+        $.get('/member/client/' + $(this).val() + '/meals/preferences', function(data) {
```

Comment:
I will comment this, even if is already merged because I think is something important and we should to fix it.
We should never use hard-coded URL like this : 
`'/member/client/' + $(this).val() + '/meals/preferences'`
If for some reason this route change this functionnality will be broken.
A workaround will be to use an attribute in the template something like `data-url` and use the Django tools to find out the proper URL using the route name.
There is an example of this in `src/fronted/js/member.js`